bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/OpenThermGatewayCallback.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Arjen Korevaar - Initial contribution
+ */
+@NonNullByDefault
+public interface OpenThermGatewayCallback {
+    public void connecting();
+
+    public void connected();
+
+    public void disconnected();
+
+    public void receiveMessage(Message message);
+
+    public void log(LogLevel loglevel, String message);
+
+    public void log(LogLevel loglevel, String format, String arg);
+
+    public void log(LogLevel loglevel, String format, Throwable t);
```

Comment:
Remove, see other comment.

Reply:
I'd like to resolve this comment to keep a clean overview of work to be done, please see my other comment on logging from the thread. Once I get to test that, I will remove the logging callback methods alltogether.